src/index.html

```diff
@@ -11,7 +11,155 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="input">
+          <label>
+            Surname:
+            <input
+              type="text"
+              name="lastName"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="input">
+          <label>
+            Name:
+            <input
+              type="text"
+              name="firstName"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="input">
+          <label>
+            How old are You?
+            <input
+              type="number"
+              name="age"
+              min="1"
+              max="100"
+              value="12"
+            >
+          </label>
+        </div>
+        <div class="input">
+          <label>
+            Full date of birth:
+            <input type="date" name="birth">
+          </label>
+        </div>
+        <div class="input">
+          <label>
+            I accept the term of the agreement
+            <input type="checkbox" name="term">
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <div class="input">
+          <label>
+            E-mail:
+            <input
+              type="email"
+              name="email"
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+        </div>
+        <div class="input">
+          <label>
+            Password:
+            <input
+              type="password"
+              name="password"
+              minlength="4"
+              maxlength="16"
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div class="input">
+          <label>
```

Comment:
leave labels only for inputs, coz now when I click on `Do you love cats?`, it chooses the first option `Yes`